web/js/components/timeline/timeline-controls/timescale-interval-change.js

```diff
@@ -126,95 +122,104 @@ class TimeScaleIntervalChange extends PureComponent {
     });
   }
 
+  renderTooltip = () => {
+    const { toolTipHovered, customIntervalText } = this.state;
+    const { hasSubdailyLayers } = this.props;
+    return (
+      <div
+        className="wv-tooltip"
+        style={{ display: toolTipHovered ? 'block' : 'none' }}
+      >
+        <div id="timeline-interval" className="timeline-interval">
+          <span
```

Comment:
Could be separate component with a time-interval parameter (e.g. day, month, year) to shrink the code. 🤷‍♂️

Reply:
Yeah or even just a separate function that takes a `interval` param to DRY this up... I just moved this existing code but will keep this in mind next time I touch this file.